### PR TITLE
Node config validator

### LINKED ISSUE #54: Outdated NodeJS config file causes unresponsive app
An outdated NodeJS config file causes an unresponsive app.
A solution would be validation of this config file, and adding missing values so the program doesn't crash.

### PR DESCRIPTION
**Pull request by jochem725**
I've added a validator to the NodeJS config, so the program no longer crashes on an invalid configuration, but uses the default. Everything is logged in the log file.

This should fix #54

### CODE REVIEW

File: frontend/modules/config.js [hunk 1]
```diff
@@ -19,6 +19,35 @@ var Config = function() {
 };
 
 /**
+ * Validates the config file keys. If one of the default keys is missing or there is a syntax error, it returns false.
+ * @param config   Config object.
+ * @returns {*}
```
Comment (bvanwijk) [inline frontend/modules/config.js]: This returns a boolean right?

**Comment by bvanwijk** [comment]
Looks good, nice that you have tested this very well.

File: frontend/modules/config.js (hunk 1 above)
Comment (jochem725) [inline frontend/modules/config.js]: Nice find, I've fixed it.

**Comment by Kingdorian** [comment]
Looks great!

**Comment by jochem725** [comment]
I'll merge it then.

**Pull request merged**